installing/installing_bare_metal_ipi/ipi-install-overview.adoc

```diff
@@ -3,12 +3,57 @@
 include::modules/common-attributes.adoc[]
 :context: ipi-install
 
+ifdef::watermark[]
+[IMPORTANT]
+====
+The Bare Metal IPI images and code described in this document are for *Developer Preview* purposes and are *not supported* by Red Hat at this time.
```

Comment:
You are correct generally. I'm going to leave it as is, because it's conditional and only appears in the upstream and it mentions developer preview, not technology preview. I'm thinking to remove the whole thing though as this may be back from 4.3 days. I'll investigate further.